### PR TITLE
[Docs] Update text (Hugging Face into OpenAI) #5003

### PR DESCRIPTION
**Pull request by benw10-1**
## Description

**Fixes** [#5003](https://github.com/mindsdb/mindsdb/issues/5003)

## Type of change

Documentation Update

### What is the solution?

Opened a text editor, then changed the requested text from `Here are the tasks supported by MindsDB and Hugging Face:` to `Here are the tasks supported by MindsDB and OpenAI:`

## Checklist:

- [ N/A ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ N/A ] I have commented my code, particularly in hard-to-understand areas.
- [   Y   ] I have updated the documentation, or created issues to update them.
- [ N/A ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [   Y   ] I have shared a short loom video or screenshots demonstrating any new functionality.
change-text-to-openai

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@benw10-1 
Before we can merge your PR, please sign our CLA following the instructions above.

Please see the review comment above and implement the changes in this PR.

**Comment by benw10-1** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/nlp/nlp-extended-examples.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ If you have a specific task or model in mind, please let us know in the [MindsDB
 
 ## Hugging Face Examples
```
[the remark this answers is not in the document]
Comment (benw10-1) [reply docs/nlp/nlp-extended-examples.mdx]: I should have checked did not realize there were 2 instances of this text :(.

**Pull request merged**